settings/prod.py

```diff
@@ -53,3 +53,6 @@
 REST_FRAMEWORK_DOCS = {
     'HIDE_DOCS': True
 }
+
+# Port number for the python-memcached cache backend.
+CACHES['default']['LOCATION'] = os.environ.get("LOCATION", "") # noqa: ignore=F405
```

Comment:
It is to fix the travis issues in order to ignore the `*` imports. As we imported `CACHES` from `common.py` using the `*`.